Hummingbird.docc/Tutorials/FluentUniverse/Fluent.tutorial

```diff
@@ -0,0 +1,11 @@
+@Tutorials(name: "Galaxy Backend") {
```

Comment:
Fluent Tutorial?
Galaxy Backend doesn't really tell us what the tutorial is about